### DISCUSSION TITLE
saf-training/courses/profile-dev-test/23

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#76] [Ideas]
# saf-training/courses/profile-dev-test/23

Understanding the kitchen.container.yml (./kitchen.container.yml) The kitchen.container.yml file orchestrates our container-based test suite. It defines two types of containers,...

https://mitre.github.io/saf-training/courses/profile-dev-test/23.html

**Comment by seanlongcc** [comment]
Having this file be right after or in the *Docker Testing Suite* guide would be helpful as I wanted to know what was happening, but this file was a bit hidden.
